data/net/minecraft/client/renderer/block/model/ItemTransform.mapping

```diff
@@ -1,7 +1,13 @@
 CLASS net/minecraft/client/renderer/block/model/ItemTransform
+	METHOD <init> (Lcom/mojang/math/Vector3f;Lcom/mojang/math/Vector3f;Lcom/mojang/math/Vector3f;)V
+		ARG 1 rotation
+		ARG 2 translation
+		ARG 3 scale
 	METHOD apply (ZLcom/mojang/blaze3d/vertex/PoseStack;)V
 		ARG 1 leftHand
 		ARG 2 matrixStack
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
```

Comment:
See comment about `other` in `equals` above. Again I don't think it's required but generally recommended.